src/details/ArborX_AccessTraits.hpp

```diff
@@ -194,10 +194,12 @@ class AccessValues
 {
 private:
   using Access = AccessTraits<Values, Tag>;
-
-public:
   Values _values;
 
+public:
+  explicit AccessValues(Values values)
+      : _values(std::move(values))
+  {}
```

Comment:
This constructor was added as compared to the version I expect you reviewed.

Reply:
Why was it not in the original PR? Did it pass without somehow?